plugin.json

```
@@ -35,7 +35,7 @@
         "key": "RolesAllowedToEditJiraSubscriptions",
         "display_name": "Mattermost Roles Allowed to Edit Jira Subscriptions",
         "type": "radio",
-        "help_text": "Mattermost users that can subscribe channels to Jira tickets.",
+        "help_text": "Mattermost users who can subscribe channels to Jira tickets. \n**Note:** Channel Admins are those given the “Manage Channel Settings” permission in the active Permissions Scheme. They “can rename channels and edit channel header and purposes” (see [the documentation](https://docs.mattermost.com/deployment/advanced-permissions.html#public-and-private-channel-management) for more info).",
```

Users with the Team Edition of MM won't have the permission scheme options in the admin console, right? A portion of this help text won't apply to them.

---

By default (in team edition) will users be able to "Manage Channel Settings"? If so, I think we should change the text to reflect that, and then the exception that they can trim permissions if they have EE.  

```suggestion
        "help_text": "Mattermost users who can subscribe channels to Jira tickets. \n**Note:** In Team edition, everyone is considered a Channel Admin and can edit the jira subscription.  In Enterprise Editions, Channel Admins are those given the “Manage Channel Settings” permission in the active Permissions Scheme. They “can rename channels and edit channel header and purposes” (see [the documentation](https://docs.mattermost.com/deployment/advanced-permissions.html#public-and-private-channel-management) for more info).",
```